subcommand/server-acl-init/command.go

```diff
@@ -491,6 +501,108 @@ func (c *Command) Run(args []string) int {
 		}
 	}
 
+	if len(c.flagIngressGatewayNames) > 0 {
+		// Create a token for each ingress gateway name. Each gateway needs a
+		// separate token because users may need to attach different policies
+		// to each gateway token depending on what the services it represents
+		for _, name := range c.flagIngressGatewayNames {
+			if name == "" {
+				c.log.Error("Ingress gateway names cannot be empty")
+				return 1
+			}
+
+			// Parse optional namespace. This does not protect against a user
```

Comment:
Did you want to give an error if they have a `.` but namespaces aren't enabled? I'm not sure where it would break in this case later down the line, if it would at all so might be nice to save someone.

Reply:
Added.